src/main/java/pi2schema/crypto/providers/DecryptingMaterialsProvider.java

```diff
@@ -1,14 +1,15 @@
 package pi2schema.crypto.providers;
 
-import pi2schema.crypto.materials.DecryptingMaterial;
 import org.jetbrains.annotations.NotNull;
 import pi2schema.crypto.materials.SymmetricMaterial;
 
+import java.io.Closeable;
 import java.util.concurrent.CompletableFuture;
 
-public interface DecryptingMaterialsProvider {
+public interface DecryptingMaterialsProvider extends Closeable {
```

Comment:
Why adding `Closeable` and a `default close` that doesn't do anything?

Reply:
So that only required implementations (like kafka which has state and open producers for instance) overwrite and do its magic without forcing to change all the other implementations (which i guess in generall will be stateless http clients in case of aws/gcp or the in memory tests).

If i am not mixing up, even the default kafka serializer and interceptor interfaces follow the same strategy.